packages/ckeditor5-engine/tests/view/renderer.js

```diff
@@ -305,6 +305,33 @@ describe( 'Renderer', () => {
 			expect( domRoot.childNodes[ 0 ] ).to.equal( domImg );
 		} );
 
+		it( 'should remove any comment from the DOM element', () => {
+			// This comment should be cleared during render.
+			const domComment = document.createComment( 'some comment from the user-land' );
+			domRoot.appendChild( domComment );
+
+			renderer.markToSync( 'children', viewRoot );
+			renderer.render();
+
+			expect( domRoot.childNodes.length ).to.equal( 0 );
+		} );
+
+		// https://github.com/ckeditor/ckeditor5/issues/5734
+		it( 'should replace a comment with the added element', () => {
```

Comment:
TBH I don't understand this description even though I read the test and I know what the issue is about 😛

Reply:
Do you have any alternative in mind?
Is "should remove the comment and add a child element" any better?

Reply:
Nothing particular in my mind. The alternative sounds better, though.